jira.sgnl.yaml

```diff
@@ -0,0 +1,164 @@
+# Jira Configuration YAML for DS2.0
+displayName: "Jira"
+icon: PHN2ZyB3aWR0aD0iNjQiIGhlaWdodD0iNjQiIHZpZXdCb3g9IjAgMCA2NCA2NCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3Qgd2lkdGg9IjY0IiBoZWlnaHQ9IjY0IiBmaWxsPSJ3aGl0ZSIvPgo8cGF0aCBkPSJNNTYuMjA4IDMwLjgzODVMMzQuNTAwMyA5LjEzMDgyTDMyLjQwOTQgN0wxNi4wNTU0IDIzLjM0MDdMOC41ODQyMiAzMC44Mzg1QzguMjEwMTIgMzEuMjEzMSA4IDMxLjcyMDggOCAzMi4yNTAyQzggMzIuNzc5NiA4LjIxMDEyIDMzLjI4NzMgOC41ODQyMiAzMy42NjE5TDIzLjUxMzMgNDguNTkwOUwzMi40MDk0IDU3LjQ3MzhMNDguNzUwMSA0MS4xMzMxTDQ5LjAwMzIgNDAuODhMNTYuMjA4IDMzLjYzNTJDNTYuNTc0IDMzLjI2MTggNTYuNzc5IDMyLjc1OTggNTYuNzc5IDMyLjIzNjlDNTYuNzc5IDMxLjcxNCA1Ni41NzQgMzEuMjEyIDU2LjIwOCAzMC44Mzg1Wk0zMi40MDk0IDM5LjY4MTRMMjQuOTUxNiAzMi4yMjM2TDMyLjQwOTQgMjQuNzY1N0wzOS44NjczIDMyLjIyMzZMMzIuNDA5NCAzOS42ODE0WiIgZmlsbD0iIzAwODJGRiIvPgo8cGF0aCBkPSJNMzIuNDA5NSAyNC43NjU4QzMwLjA2MzkgMjIuNDE5OSAyOC43NDE3IDE5LjI0MTIgMjguNzMxNyAxNS45MjM5QzI4LjcyMTggMTIuNjA2NSAzMC4wMjQ3IDkuNDE5OTUgMzIuMzU2MiA3LjA2TDE2LjA0MjEgMjMuMzc0MUwyNC45MjUgMzIuMjU2OUwzMi40MDk1IDI0Ljc2NThaIiBmaWxsPSJ1cmwoI3BhaW50MF9saW5lYXJfMTIxNV83ODApIi8+CjxwYXRoIGQ9Ik0zOS44NzQxIDMyLjIwMzZMMzIuNDA5NSAzOS42ODE1QzMzLjU4MDUgNDAuODQ4MiAzNC41MDk2IDQyLjIzNDYgMzUuMTQzNSA0My43NjEyQzM1Ljc3NzUgNDUuMjg3OSAzNi4xMDM4IDQ2LjkyNDYgMzYuMTAzOCA0OC41Nzc2QzM2LjEwMzggNTAuMjMwNiAzNS43Nzc1IDUxLjg2NzQgMzUuMTQzNSA1My4zOTRDMzQuNTA5NiA1NC45MjA2IDMzLjU4MDUgNTYuMzA3IDMyLjQwOTUgNTcuNDczOEw0OC43NzAyIDQxLjExMzFMMzkuODc0MSAzMi4yMDM2WiIgZmlsbD0idXJsKCNwYWludDFfbGluZWFyXzEyMTVfNzgwKSIvPgo8ZGVmcz4KPGxpbmVhckdyYWRpZW50IGlkPSJwYWludDBfbGluZWFyXzEyMTVfNzgwIiB4MT0iMzEuMDY0NCIgeTE9IjE3LjIyMTMiIHgyPSIyMC42NSIgeTI9IjI3LjYzNTciIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj4KPHN0b3Agb2Zmc2V0PSIwLjE4IiBzdG9wLWNvbG9yPSIjMDA1MEQzIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iIzAwODJGRiIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MV9saW5lYXJfMTIxNV83ODAiIHgxPSIzMy44MjEyIiB5MT0iNDcuMTM5MyIgeDI9IjQ0LjIxNTYiIHkyPSIzNi43NDQ5IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIG9mZnNldD0iMC4xOCIgc3RvcC1jb2xvcj0iIzAwNTBEMyIvPgo8c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiMwMDgyRkYiLz4KPC9saW5lYXJHcmFkaWVudD4KPC9kZWZzPgo8L3N2Zz4K
+description: "Jira as a System of Record"
+address: "{{Input Required: Jira-Domain}}.atlassian.net"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+adapter:
+  hostname: "adapter-sgnl.sgnl.svc.cluster.local"
+  port: 80
+  # Example Configs:
+  # empty config: {} | b64: "e30="
```

Comment:
Can you replace config with this value? We want to have the template usable with least amount of input required from the users; i.e., address, and auth..